scripts/lxd-images

```diff
@@ -91,9 +91,18 @@ class UnixHTTPConnection(http.client.HTTPConnection):
 
 
 class LXD(object):
+    workdir = None
+
     def __init__(self, path):
         self.lxd = UnixHTTPConnection(path)
 
+        # Create our workdir
+        self.workdir = tempfile.mkdtemp()
+
+    def __del__(self):
```

Comment:
So I always thought this sucked, but __del__ isn't guaranteed to ever be called. If you want cleanup behavior, you have to use a context manager and define __exit__. (Perhaps we don't care about this as it rarely happens, but it's worth noting.)